tsdb/engine/tsm1/engine_test.go

```diff
@@ -16,6 +19,7 @@ import (
 	"github.com/influxdata/influxdb/pkg/deep"
 	"github.com/influxdata/influxdb/tsdb"
 	"github.com/influxdata/influxdb/tsdb/engine/tsm1"
+	// "github.com/pkg/profile"
```

Comment:
I left the reference to the package so that others would know how to run local profiles. The built-in `-memprofile` doesn't work well since it collects the allocations during the write path too.

I'll move this to a documentation section in `CONTRIBUTING.md`